app/javascript/src/components/login/signup/When2Meet.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+
+// semantic-ui
+import { Grid, Segment, Divider, Button, Icon, Card } from 'semantic-ui-react';
+
+// helpers
+import _ from 'lodash';
+
+const times = [
+    'Night Shift',
+  '07:00 - 08:15',
+  '08:15 - 09:30',
+  '09:30 - 10:45',
+  '10:45 - 12:00',
+  '12:00 - 13:15',
+  '13:15 - 14:30',
+  '14:30 - 15:45',
+  '15:45 - 17:00',
+  '17:00 - 18:15',
+  '18:15 - 19:30',
+  '19:30 - 20:45',
+  '20:45 - 22:00',
+  '22:00 - 23:15',
+  '23:15 - 00:30',
```

Comment:
Why are we using 24 hour clock? It isn't consistent with our Calendar component and most users probably use 12-hr over 24. So go ahead and just change this all to 12 hr.

Reply:
I want to use 24 hour clock. We should talk about this.